app/models/manageiq/providers/redhat/network_manager.rb

```diff
@@ -1,14 +1,9 @@
 class ManageIQ::Providers::Redhat::NetworkManager < ManageIQ::Providers::NetworkManager
-  require_nested :CloudNetwork
-  require_nested :CloudSubnet
   require_nested :EventCatcher
   require_nested :EventParser
   require_nested :FloatingIp
```

Comment:
I didn't have to delete this for the specs to pass, but I don't see why this wouldn't have the same issue.  Investigating...

Reply:
Looks like there just weren't any floating_ips in this VCR, still need to delete this